internal/extproc/translator/openai_gcpvertexai_test.go

```diff
@@ -427,12 +479,41 @@ func TestOpenAIToGCPVertexAITranslatorV1ChatCompletion_ResponseBody(t *testing.T
 			},
 			wantTokenUsage: LLMTokenUsage{},
 		},
+		{
+			name: "single stream chunk response",
```

Comment:
i can't find multiple chunks test case. take a look at TestOpenAIToAWSBedrockTranslatorV1ChatCompletion_Streaming_ResponseBody and let's have a similar one